shared/desktop/remote/store.desktop.js

```diff
@@ -12,12 +12,14 @@ class RemoteStore {
   _internalState: any = {}
   _gotPropsCallback: ?() => void // let component know it loaded once so it can show itself. Set to null after calling once
 
-  _onPropsUpdated = props => {
+  _onPropsUpdated = propsStr => {
+    const props = JSON.parse(propsStr)
```

Comment:
Looking at the rpc traffic between the windows sending a plain object actually creates a ton of overhead. it tries to 'type' each property which is a huge multiple. Instead we send a string over